modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -96,6 +96,8 @@ def make_request(endpoint:, action:, body:, key:) # rubocop:disable Metrics/Meth
                           'Soapaction' => "\"#{action}\""
                         })
       end
+      raise Faraday::ServerError, 'Internal Server Error' if response.status == 500 && action == 'insertIntentToFile'
```

Comment:
Will this request always respond with a 500, or should we capture all >= 500 errors as a ServerError? Should 400s be included too (rare though they may be)?

Since BGS communication has been giving us some trouble lately, let's also log (not necessarily duration log) the bad statuses we receive and why the exception is getting thrown.

Reply:
Updated to catch all >= 500 errors and log the specific HTTP status and reason phrase returned via the server response. 

I opted against catching all HTTP 400s at this point, but I can update to handle those if you feel it's warranted.